### DISCUSSION TITLE
tutorial/use/initialize

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#12] [General]
# tutorial/use/initialize

http://localhost:5173/tutorial/use/initialize

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by long36708** [comment]
useState 的初始化机制 有没有具体的介绍？

**Reply by hangYq** [reply]
https://zh-hans.react.dev/reference/react/useState#usestate
https://zh-hans.react.dev/reference/react/useState#avoiding-recreating-the-initial-state
